pages/index.tsx

```diff
@@ -1,8 +1,19 @@
 import Head from 'next/head'
+import {
+  PageContainer
+} from './styles';
 import styles from '../styles/Home.module.css'
 import firebase from '../firebase/clientApp'
+<<<<<<< HEAD
```

Comment:
notes for the globals.css file in general. before you push anything, always remember to remove any conflicting elements of your branch. in this case it would be the arrows <<<<<< HEAD